src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -183,8 +297,9 @@ const CollectionItem = ({
       style={{
         paddingBottom: '22px',
         paddingTop: '22px',
+        ...style,
       }}
-      maxHeight="44"
+      maxHeight={`${COLLECTION_ROW_HEIGHT}`}
```

Comment:
nit: Can you just do `{COLLECTION_ROW_HEIGHT}`

Reply:
nah it needs to be a string